ReactiveCocoa/Swift/Property.swift

```diff
@@ -95,6 +95,57 @@ extension MutableProperty: SinkType {
 	}
 }
 
+/// Wraps a `dynamic` property, or one defined in Objective-C, using Key-Value
+/// Coding and Key-Value Observing.
+///
+/// Use this class only as a last resort! `MutableProperty` is generally better
+/// unless KVC/KVO is required by the API you're using (for example,
+/// `NSOperation`).
+@objc public final class DynamicProperty: NSObject, PropertyType {
+	public typealias Value = AnyObject?
```

Comment:
Oh, I didnt even consider Obj-C compatibility. We could always have a subclass of this just for swift perhaps (`DynamicPropertyOf<T>` or something). We would cast, but that would be an implementation detail, this API would be nicer to use  (when working with the property's producer)